### PR TITLE
Settings branch

### PR DESCRIPTION
**Pull request by wallacethierre**

### CODE REVIEW

File: drinkingfountain/src/main/java/sample/knot/cesar/org/br/drinkingfountain/ui/SettingsActivity.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package sample.knot.cesar.org.br.drinkingfountain.ui;
```
Comment (emidiooliveira) [inline drinkingfountain/src/main/java/sample/knot/cesar/org/br/drinkingfountain/ui/SettingsActivity.java:11]: Did you put the cesar's header?

File: drinkingfountain/src/main/java/sample/knot/cesar/org/br/drinkingfountain/ui/SettingsActivity.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    };
+
+    private void storeInformation() {
+        PreferenceUtil.getInstance().setEndPoint(mEdtEndPoint.getText().toString());
+        PreferenceUtil.getInstance().setUuid(mEdtUUID.getText().toString());
+        PreferenceUtil.getInstance().setToken(mEdtToken.getText().toString());
+    }
+
+    private boolean iSCanSave() {
+        Log.i("DJACA", "Primeira expressão: "+ (mEdtEndPoint != null &&  !mEdtEndPoint.getText().toString().equals(Util.EMPTY_STRING) ));
```
Comment (emidiooliveira) [inline drinkingfountain/src/main/java/sample/knot/cesar/org/br/drinkingfountain/ui/SettingsActivity.java]: Remove these logs

File: drinkingfountain/src/main/java/sample/knot/cesar/org/br/drinkingfountain/ui/SettingsActivity.java (hunk 1 above)
Comment (wallacethierre) [reply drinkingfountain/src/main/java/sample/knot/cesar/org/br/drinkingfountain/ui/SettingsActivity.java:11]: Done!

File: drinkingfountain/src/main/java/sample/knot/cesar/org/br/drinkingfountain/ui/SettingsActivity.java (hunk 2 above)
Comment (wallacethierre) [reply drinkingfountain/src/main/java/sample/knot/cesar/org/br/drinkingfountain/ui/SettingsActivity.java]: Done!

**Pull request merged**